.ci/e2eKibana.groovy

```diff
@@ -3,7 +3,7 @@
 @Library('apm@current') _
 
 pipeline {
-  agent none
+  agent { label 'ubuntu-20' }
```

Comment:
I'd say to leave none and use the agent in the stage itself

Reply:
I added a commit on top with your suggestion, thanks!